Formula/shapelib.rb

```diff
@@ -20,8 +21,13 @@ class Shapelib < Formula
   depends_on "cmake" => :build
 
   def install
-    system "cmake", ".", *std_cmake_args
-    system "make", "install"
+    # shapelib's CMake scripts interpret `CMAKE_INSTALL_LIBDIR=lib` as relative
+    # to the current directory, i.e. `CMAKE_INSTALL_LIBDIR:PATH=$(pwd)/lib`
+    args = std_cmake_args.reject { |s| s["CMAKE_INSTALL_LIBDIR"] }
+    mkdir "build" do
+      system "cmake", "..", *args, "-DCMAKE_INSTALL_LIBDIR=#{lib}"
```

Comment:
```suggestion
    mkdir "build" do
      system "cmake", "..", *std_cmake_args, "-DCMAKE_INSTALL_LIBDIR=#{lib}"
```
This works too, since `cmake` will take the later value of `CMAKE_INSTALL_LIBDIR`.

Reply:
I think we usually remove duplicates based on other formulae.
From formula simplicity, maybe just relying on `cmake` behavior would be cleaner.
Though, from output logs, removing the duplicate would be cleaner.

Reply:
We do remove duplicates in other formulae, but I suspect that more likely stems from a lack of familiarity with how CMake treats duplicate keys in command-line arguments than anything else. (Compare with `include-what-you-use`, which was contributed by a CMake maintainer.)

Leaving it this way is fine with me too.

Reply:
Maybe the better way is to switch to using an argument?
```rb
  def std_cmake_args(install_prefix: prefix, install_libdir: "lib", find_framework: "LAST")
    args = %W[
      -DCMAKE_INSTALL_PREFIX=#{install_prefix}
      -DCMAKE_INSTALL_LIBDIR=#{install_libdir}
```

Reply:
We can do that, but we need to wait until Homebrew/brew@12435565f3c1aeb86d62a366a4e1e1878e99153f lands in a `brew` release tag.

Reply:
Sure. We can wait for next brew tag. Updated PR with changes and can rerun CI whenever available.